src/modules/DOM.js

```diff
@@ -0,0 +1,41 @@
+import Countries from './rest_countries.js';
+
+class DOM {
+  constructor() {
+    this.countries = new Countries();
+    this.container = document.querySelector('.wrapper');
+  }
+
+  createHtmlContent = async (country) => {
+    const countryInfo = this.countries.getCountryShortInfo(country);
+    const html = `
+    <article class="wrapper__card">
+      <img class="wrapper__card__img" src="${
+        (await countryInfo).flagURl
+      }" alt="${(await countryInfo).name}" />
+      <div class="wrapper__card__details">
+        <h3 class="wrapper__card__details__title">${
+          (await countryInfo).name
+        }</h3>
+        <div class="wrapper__card__details__likes">
+          <button class="wrapper__card__details__likes-btn"><i class="bi bi-heart-fill"></i></button>
+          <p></p>
+        </div>
+      </div>
+      <button class="wrapper__card__comments-btn card__btn">Comments</button>
```

Comment:
JS classes are widely recommended for the modularization of the code. So well done using the async and await method is helpful to render the web app content.